hecate.py

```diff
@@ -269,13 +329,57 @@ def getConfig(self, key):
                                  'config file, or set hecate_%s' % (key, key))
         return value
 
+    def cloud_container_add(self, container_name):
+        '''Method to create container in cloud.'''
+        toReturn = ''
+        provider = self.getConfig('provider')
+        headers = self.getConfig('headers')
+        if provider.lower() == 'rackspace':
+            # supports Rackspace Openstack on auth v2
+            self._rs_openstack_auth_helper(headers)
+            toReturn = self._openstack_conatiner_helper(container_name, headers)
+        elif provider.lower() == 'openstack':
+            self._openstack_auth_helper(headers)
+            toReturn = self._openstack_conatiner_helper(container_name, headers)
+            # supports Openstack on auth v3
+        else:
+            # the developer is using Rackspace Openstack; if others wish to
+            # write their own authentication helpers, pull requests are
+            # accepted.
+            raise NotImplementedError('The provider'
+                                      ' %s is not available.' % provider)
+        return toReturn
+
+
+    def cloud_container_remove(self, container_name):
+        '''Method to remove container in cloud.'''
+        toReturn = ''
+        provider = self.getConfig('provider')
+        headers = self.getConfig('headers')
+        if provider.lower() == 'rackspace':
+            # supports Rackspace Openstack on auth v2
+            self._rs_openstack_auth_helper(headers)
+            toReturn = self._openstack_conatiner_helper(container_name, headers,
+                remove=True)
+        elif provider.lower() == 'openstack':
+            self._openstack_auth_helper(headers)
+            toReturn = self._openstack_conatiner_helper(container_name, headers,
+                remove=True)
+            # supports Openstack on auth v3
+        else:
+            # the developer is using Rackspace Openstack; if others wish to
+            # write their own authentication helpers, pull requests are
+            # accepted.
+            raise NotImplementedError('The provider'
+                                      ' %s is not available.' % provider)
+        return toReturn
 
 def process(arguments):
     '''Takes the command line arguments and calls the appropriate Hecate
      methods.'''
-    if not arguments.file:
-        return 'No files chosen, use -f, --file to specify filenames (space '\
-               'delineated).'
+    if not arguments.file and not arguments.remove and not arguments.newContainer \
```

Comment:
couldn't this better be written if not (arguments.file or arguments.remove or ...)? Might be a little more clear.